tests/tests/test-precompile/test-precompile-xcm-utils.ts

```diff
@@ -0,0 +1,123 @@
+import "@moonbeam-network/api-augment";
+
+import { u8aToHex } from "@polkadot/util";
+import { expect } from "chai";
+import { ethers } from "ethers";
+import { PRECOMPILE_XCM_UTILS_ADDRESS } from "../../util/constants";
+import { getCompiled } from "../../util/contracts";
+import { web3EthCall } from "../../util/providers";
+import { describeDevMoonbeamAllEthTxTypes, DevTestContext } from "../../util/setup-dev-tests";
+
+import { descendOriginFromAddress } from "../../util/xcm";
+
+const XCM_UTILS_CONTRACT = getCompiled("XcmUtils");
+const XCM_UTILSTRANSACTOR_INTERFACE = new ethers.utils.Interface(XCM_UTILS_CONTRACT.contract.abi);
+
+describeDevMoonbeamAllEthTxTypes("Precompiles - xcm UTILS", (context) => {
```

Comment:
Any specific reason for the all-caps on `UTILS` ?

Reply:
nope